example/lib/src/controls.dart

```diff
@@ -90,7 +90,7 @@ class _ControlsState extends State<Controls> {
     }
 
     try {
-      await track.restartTrack(LocalVideoTrackOptions(position: position));
+      await track.restartTrack(options: LocalVideoTrackOptions(position: position));
```

Comment:
what's the trade off with the named parameter? IMO with a single optional param, it seems simpler to make it positional. Though I'm open to alternative views.

Reply:
Fixed 88d04bfeff745e5217673d7474190863ee28f4a9 !